web/app.js

```diff
@@ -2029,6 +2044,15 @@ const PDFViewerApplication = {
       _windowAbortController: { signal },
     } = this;
 
+    this._touchManager = new TouchManager({
+      container: window,
+      isPinchingDisabled: () => this.pdfViewer.isInPresentationMode,
+      isPinchingAllowed: () => this.overlayManager?.active,
```

Comment:
I might be misunderstanding your intent here, but this name feels "inverted" to me since the code is preventing touch-zooming from occurring when the `overlayManager` is active.

Naming things is always difficult, but would e.g. the following make more sense?
```suggestion
      isPinchingStopped: () => this.overlayManager?.active,
```